test/org/zaproxy/zap/testutils/websocket/server/NanoWebSocketTestServer.java

```diff
@@ -37,6 +37,8 @@
     private final static String CLOSE_REASON_CANT_SENT = "Connection Lost";
     private final static String CLOSE_REASON_REQUIREMENT = "Requirement";
     
+    private boolean isSecure = false;
```

Comment:
This may seem odd but will help if we decide later to establish secure WebSocket connection

Reply:
Looks good to me.